src/Compiler/Microsoft.NET.Sdk.Razor.SourceGenerators/SourceGeneratorProjectEngine.cs

```diff
@@ -0,0 +1,115 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#nullable enable
+
+using Microsoft.AspNetCore.Razor.Language;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+
+namespace Microsoft.NET.Sdk.Razor.SourceGenerators;
+
+internal class SourceGeneratorProjectEngine : DefaultRazorProjectEngine
+{
+	private readonly int discoveryPhaseIndex = -1;
+
+	private readonly int rewritePhaseIndex = -1;
+
+	public SourceGeneratorProjectEngine(DefaultRazorProjectEngine projectEngine)
+		: base(projectEngine.Configuration, projectEngine.Engine, projectEngine.FileSystem, projectEngine.ProjectFeatures)
+    {
```

Comment:
You seem to be using tabs somewhere and spaces elsewhere (not just in this file).

Reply:
Ugh. Looks like my VS settings got all messed up. Fixing.